pwndbg/commands/heap.py

```diff
@@ -81,7 +81,13 @@ def format_bin(bins: Bins, verbose=False, offset=None):
             formatted_chain = pwndbg.chain.format(chain_fd[0], offset=offset, safe_linking=safe_lnk)
 
         if isinstance(size, int):
-            size = hex(size)
+            if bins_type == BinType.LARGE:
+                infinity_symbol = "\u221e"  # Unicode "infinity"
+                start_size, end_size = allocator.largebin_size_range_from_index(size)
+                size = hex(start_size) + "-"
+                size += infinity_symbol if end_size == pwndbg.gdblib.arch.ptrmask else hex(end_size)
```

Comment:
If we put the comment for infinity symbol anyway, we can just hardcode it here instead of storing in a variable

Reply:
I've split this into a a multi-line `if`/`else`, otherwise the linter makes it harder to read.